roles/matrix-synapse/defaults/main.yml

```diff
@@ -286,6 +298,35 @@ matrix_synapse_metrics_port: 9100
 # See https://github.com/matrix-org/synapse/blob/master/docs/manhole.md
 matrix_synapse_manhole_enabled: false
 
+# Enable support for Synapse workers
+matrix_synapse_workers_enabled: false
+
+# Default list of workers to spawn (order in accord to docs)
+# - no endpoints / doesn't need port mapping if port ends on 0
+# - single-instance-only if 2nd last digit of port number is 0
+matrix_synapse_workers_enabled_list:
+  - { type: generic_worker, port: 18111, metrics_port: 19111 }
+  - { type: generic_worker, port: 18112, metrics_port: 19112 }
+  - { type: generic_worker, port: 18113, metrics_port: 19113 }
+  - { type: generic_worker, port: 18114, metrics_port: 19114 }
+  - { type: generic_worker, port: 18115, metrics_port: 19115 }
+  - { type: generic_worker, port: 18116, metrics_port: 19116 }
+  - { type: pusher, port: 00, metrics_port: 19200 }
```

Comment:
From what I could find in the code, we don't make a distinction between `00` and `0` (but I may have missed it).

If that's really the case, we should probably just annotate these `00` entries with some comment and turn them into a `0`.

If we do need to make some distinction, it's probably better to add more fields (`multiple_instances_supported: false`) and use those.

What do you think?